api/app/Http/Controllers/ChildEntityController.php

```diff
@@ -224,7 +224,7 @@ public function deleteOne($id, $childId)
         $model = $this->findParentEntity($id);
         $childModel = $this->findOrFailChildEntity($childId, $model);
 
-        $childModel->delete();
+        $model->deleteChild($childModel, $this->relationName);
```

Comment:
@artstorm if we just need to raise an event for the purposes of the revisionable trait, we should probably just do that explicitly, otherwise it isn't clear that the deleteChild method actually is overridden by the trait.